pkg/util/errutil/status.go

```diff
@@ -37,15 +37,24 @@ const (
 	// an error, but that there is nothing the client can do to fix it.
 	// HTTP status code 500.
 	StatusInternal CoreStatus = "Internal server error"
-	// StatusTimeout means that the server did not complete the request
+	// StatusRequestTimeout means that the server did not complete the request
 	// within the required time and aborted the action.
-	// HTTP status code 504.
-	StatusTimeout CoreStatus = "Timeout"
+	// HTTP status code 408.
```

Comment:
If the server cannot process the request in time it should be an 5xx and threat as an error. 

Datasources should as well. we just don't want to threat it as an error in our SLO :)

408 is about the client not sending its data (ex post body) fast enough. 

```
HTTP 408 status code is returned when a server timeout occurs due to a slow client request. It means the server terminated the connection, resulting in the client receiving the 408 Request Timeout message.
```

Reply:
Thanks make sense. Then I'll remove the StatusRequestTimeout and related since not used anywhere